docusaurus.config.js

```diff
@@ -335,11 +335,16 @@ const config = {
             path: 'next',
             banner: 'unreleased',
           },
-          3.25: {
-            label: '3.25 (latest)',
+          3.26: {
```

Comment:
If we're just cutting the version, and not publishing:
* add onlyIncludeVersions variable
* Keep 3.25 as-is (latest)
* 3.36 needs path: 3.26